tests/owncloud-sign-url/signUrlTest.js

```diff
@@ -6,7 +6,7 @@ const HTTP_GET_METHOD = 'get'
 const HTTP_POST_METHOD = 'post'
 const TEST_PORT = 33001
 
-describe('SignUrl tests', () => {
+fdescribe('SignUrl tests', () => {
```

Comment:
confusing - title states 'enable tests' but the test is disabled?

Reply:
the `f` enables the test. As we are in the process of changing tests over to pact.io tests that should run have an `f` infront of `describe` when all tests are converted all the `f`s will be deleted